kubectl-mcp-app/README.md

```
@@ -0,0 +1,372 @@
+    },
+    "kubectl-app": {
+      "command": "npx",
+      "args": ["kubectl-mcp-app", "--backend", "http://localhost:8000/mcp"]
+    }
+  }
+}
+```
+
+## CLI Options
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)